### PR TITLE
Remove thread option for now in overlap code

### PR DESCRIPTION
**Pull request by JustinChu**

### CODE REVIEW

**Comment by JustinChu** [comment]
The version of clang-format on my machine doesn't format the same as the on on azure. I feel like this check might be a little strigent

**Comment by JustinChu** [comment]
Version:
`clang-format version 3.8.0-2ubuntu4 (tags/RELEASE_380/final)`

**Comment by sjackman** [comment]
Running the same version of `clang-format` on your local machine should make all the needed changes for you.

**Comment by sjackman** [comment]
We could even create a `make` target to run `clang-format` and reformat the C++ code.

File: src/physlr-overlap.cc [hunk 1]
```diff
@@ -52,7 +52,7 @@ printHelpDialog()
 	    "Usage: physlr-overlap [OPTION]... [MINIZERS.tsv]\n"
 	    "Read a sketch of linked reads and find overlapping barcodes.\n"
 	    "  -n, --min-n=INT   Remove edges with fewer than n shared markers [0].\n"
-	    "  -t, --threads=INT Number of threads [1].\n"
+		//"  -t, --threads=INT Number of threads [1].\n"
```
Comment (sjackman) [inline src/physlr-overlap.cc]:
```suggestion
```
Delete this line.

File: src/physlr-overlap.cc [hunk 2]
```diff
@@ -108,21 +108,21 @@ main(int argc, char* argv[])
 
 	// long form arguments
 	static struct option long_options[] = { { "min-n", required_argument, nullptr, 'n' },
-		                                    { "threads", required_argument, nullptr, 't' },
+	//		                                    { "threads", required_argument, nullptr, 't' },
```
Comment (sjackman) [inline src/physlr-overlap.cc]: You can leave this as is.

File: src/physlr-overlap.cc [hunk 3]
```diff
@@ -108,21 +108,21 @@ main(int argc, char* argv[])
...
 		                                    { "version", no_argument, nullptr, 'v' },
 		                                    { nullptr, 0, nullptr, 0 } };
 
 	int i = 0;
-	while ((c = getopt_long(argc, argv, "n:t:v:", long_options, &i)) != -1) {
+	while ((c = getopt_long(argc, argv, "n:v:", long_options, &i)) != -1) {
```
Comment (sjackman) [inline src/physlr-overlap.cc]: You can leave this as is.

File: src/physlr-overlap.cc [hunk 4]
```diff
@@ -108,21 +108,21 @@ main(int argc, char* argv[])
...
-			break;
-		}
+		//		case 't': {
+		//			std::stringstream convert(optarg);
+		//			if (!(convert >> opt::threads)) {
+		//				std::cerr << "Error - Invalid parameters! t: " << optarg << std::endl;
+		//				return 0;
+		//			}
+		//			break;
+		//		}
```
Comment (sjackman) [inline src/physlr-overlap.cc]: You can leave this as is.

Add a check `if (opt::threads > 1) print an error message and exit(EXIT_FAILURE);`

**Comment by sjackman** [comment]
Add a check `if (opt::threads > 1) print an error message and exit(EXIT_FAILURE);`

**Comment by JustinChu** [comment]
I'm having trouble finding clang-format-7 in my sources, similar to this guy:
https://askubuntu.com/questions/1132412/how-to-install-clang-format-7-from-bionic-updates-universe-on-18-04-1-lts

**Comment by JustinChu** [comment]
This was literally supposed to take me 5 minutes, and ends up with me having to possibly update my OS...

File: src/physlr-overlap.cc [hunk 5]
```diff
@@ -146,6 +145,12 @@ main(int argc, char* argv[])
 	omp_set_num_threads(opt::threads);
 #endif
 
+	//Threads currently not supported
```
Comment (sjackman) [inline src/physlr-overlap.cc]:
```suggestion
	// Threads currently not supported.
```

File: src/physlr-overlap.cc [hunk 6]
```diff
@@ -146,6 +145,12 @@ main(int argc, char* argv[])
...
+	if(opt::threads > 1){
```
Comment (sjackman) [inline src/physlr-overlap.cc]:
```suggestion
	if (opt::threads > 1) {
```

File: src/physlr-overlap.cc [hunk 7]
```diff
@@ -146,6 +145,12 @@ main(int argc, char* argv[])
...
+		std::cerr << "threads not yet supported" << std::endl;
```
Comment (sjackman) [inline src/physlr-overlap.cc]:
```suggestion
		std::cerr << "Error: physlr-overlap: Threads not yet supported.\n";
```

**Comment by sjackman** [comment]
```diff
-	//Threads currently not supported
-	if(opt::threads > 1){
+	// Threads currently not supported
+	if (opt::threads > 1) {
```
See https://dev.azure.com/bcgsc/btl/_build/results?buildId=1166

**Pull request merged**